CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ the code was deployed.
 
 ## [Unreleased]
 
+## [1.3.0] - 2020-12-22
+### Added
+- Endpoints
```

Comment:
would wanna be a bit more specific here, like
- Device health endpoint
- IM21 door sensor endpoint

Reply:
Please also include the ClickUp number.